crates/eframe/src/native/glow_integration.rs

```diff
@@ -941,7 +941,9 @@ impl GlutinWindowContext {
         // Create GL display. This may probably create a window too on most platforms. Definitely on `MS windows`. Never on Android.
         let display_builder = glutin_winit::DisplayBuilder::new()
             // we might want to expose this option to users in the future. maybe using an env var or using native_options.
-            .with_preference(glutin_winit::ApiPreference::FallbackEgl) // https://github.com/emilk/egui/issues/2520#issuecomment-1367841150
+            //
+            // The justification for FallbackEgl over PreferEgl is at https://github.com/emilk/egui/pull/2526#issuecomment-1400229576 .
+            .with_preference(glutin_winit::ApiPreference::PreferEgl)
```

Comment:
Am I misunderstanding something or should this have stayed `FallbackEgl` ?

Reply:
Yes it should have. My bad. I was playing around with this setting to understand it, which is how I ended up updating the documentation comment. I accidentally committed the change when I only wanted to commit the documentation.

Reply:
https://github.com/emilk/egui/pull/5408